packages/flutter/lib/src/material/raised_button.dart

```diff
@@ -272,6 +283,11 @@ class RaisedButton extends StatelessWidget {
   /// shape as well.
   final ShapeBorder shape;
 
+  /// Defines the duration of animated changes for [shape] and [elevation].
+  ///
+  /// The default value is [kThemeChangeDuration].
+  final Duration duration;
```

Comment:
Same concern about the name here. "duration" could mean many things on an interactive widget like a button (duration of tap? response duration? animation duration?)

Reply:
OK, I used animationDuration